### DISCUSSION TITLE
Any ideas for new examples?

### DISCUSSION THREAD

**Discussion by AlbertShown** [webui-dev/webui#182] [Ideas]
We are currently using webui 2.3.0 in two of our commercial projects. After I saw all those new commits, I decided to contribute to this new release so we could upgrade our existing projects.

I would like to start with a new API to disable a specific browser. And maybe creating new examples... any ideas?

**Comment by ttytm** [comment]
I think "real" mini projects that use WebUI would be awesome. Examples that use a framework were repeatedly asked for. So this would fit here.

E.g. I very recently created a V [webview](https://github.com/ttytm/webview) wrapper and as an example of using it with SvelteKit added https://github.com/ttytm/emoji-mart-desktop/. It works very similar to WebUI. I will prep something like this for v-webui as well. But there is still some work todo on the wrapper itself before I can start working on such an example.

So any cool little showcase that is a "real" application, so to speak, would be great. Even if it doesn't use a framework.

**Comment by hassandraga** [comment]
@ttytm I would like to add `app.w.set_size()` like you did in your [webview](https://github.com/ttytm/webview#usage-example) wrapper.
@AlbertShown I'm already working a media player example. In other hand, we need to remove all duplicated makefiles from examples.

**Comment by ttytm** [comment]
Gonna prep something nice with Vue this or next week, depending on what time allows 😇

**Reply by hassandraga** [reply]
A Calc, Text editor, Tic-tac-toe game, and video player (HTML5)... I started a new AI project using WebUI.
